rllib/core/rl_trainer/tests/test_rl_trainer.py

```diff
@@ -12,10 +12,8 @@
 from ray.rllib.utils.test_utils import check, get_cartpole_dataset_reader
 
 
-def get_trainer(scaling_config=None, distributed: bool = False) -> RLTrainer:
+def get_trainer() -> RLTrainer:
```

Comment:
let's leave distributed as an input. I was thinking of extending the unittest to loop thru distributed True, False